### PR TITLE
add buttonClickFunction

### PR DESCRIPTION
**Pull request by Jack-Vu**

### CODE REVIEW

**Review by WenjiZ — DISMISSED** [review]
LGTM!

**Review by NiluparErkin — DISMISSED** [review]

**Review by ningyang534 — DISMISSED** [review]
looks great!

**Review by Muss768 — DISMISSED** [review]
LGTM

File: src/test/java/utilities/Util.java [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+
 public class Util {
 
+	
+	/*
+	 * Clicks on buttons based on xpath of button 
+	 * @Author: Jack Vu 
+	 * 
+	 */
+	public static void buttonClickFunction(WebDriver driver, String xpathOfButton) {
```
Comment (boratech-macmini) [inline src/test/java/utilities/Util.java]: It would be a better idea to use `By` object instead of xpath, would be more reusable

**Review by boratech-macmini — CHANGES_REQUESTED** [review]

File: src/test/java/utilities/Util.java (hunk 1 above)
Comment (Jack-Vu) [reply src/test/java/utilities/Util.java]: Used By object, should be ready to go.

**Review by boratech-macmini — APPROVED** [review]

**Review by Muss768 — APPROVED** [review]
LGTM

**Pull request merged**